PractiseMay11/PlantsTask/Plant.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseMay11.PlantsTask
+{
+    internal class Plant
+    {
+        public string Type { get; }
+
+        public Color Color { get; set; }
+        
+        public int Height { get; set; }
+
+        public Plant(string type, int height)
+        {
+            this.Type = type;
+            this.Color = 0;
+            this.Height = height;
+        }
+
+        public void Drink(double waterAmount)
+        {
```

Comment:
Please don't forget to add validation for all public members to check if parameters are valid.
For example you can specify 0 water and logic will say that plant was poured with 0 water. But in this case this means nothing, so I'd avoid illegal cases.